lib/proxy/log_buffer/decorator.rb

```diff
@@ -53,6 +52,7 @@ def add(severity, message = nil, progname = nil, backtrace = nil)
           @buffer.push(rec)
         end
       end
+      debug("Reopened logging files") if reopened
```

Comment:
I would like this to show up as the very first line after reopening. Otherwise you'll end up with a log file that says `<message>` and then `Reopened logging files`. However, I'm aware this can cause (endless) recursing. Would it be better to use the raw `@buffer.push for this right after reopening?

Note I actually do like this log line and wonder if it should actually be an `info` message. As an admin it can be very useful to know when a log file was created.

Reply:
This does work as expected actually, it appears as the very first log line and it does not end up in endless loop. Let me change that to INFO level then.